BlueMapCore/src/main/java/de/bluecolored/bluemap/core/world/mca/chunk/Chunk_1_13.java

```diff
@@ -113,6 +119,10 @@ public Chunk_1_13(MCAWorld world, Data data) {
             this.sectionMin = 0;
             this.sectionMax = 0;
         }
+
+        this.blockEntities = level.blockEntities.stream().collect(Collectors.toMap(
+                it -> (long) it.getY() << 40 | (it.getX() & 0xF) << 4 | it.getZ() & 0xF, it -> it
```

Comment:
I really like this solution for quick access!
Can you explain why y is shifted by `40` tho? I would have expected `8` or maybe `32` 🤔

Reply:
You are right, it should be 8. It was late, the 40 came from 8 (x and z) + 32 (y) because for some reason I thought there that the shift moves the most significant bit. Oops.